test/pass_files/alias_assign.d

```diff
@@ -0,0 +1,14 @@
+template T(X)
+{
+     alias A = int;
```

Comment:
To add to what WebFreak has said, and being more specific, the code DScanner is blocking should be used as tests e.g. nu-staticMap etc.